docs/_documentations/importing-existing-projects.md

```diff
@@ -14,6 +14,12 @@ order: 0
 
 Modifications are usually required to successfully add and deploy projects that have never been run in Codewind before. This tutorial covers the basics of configuring a project to run in Codewind.
 
+## Requirements for importing an existing project not created in Codewind or through Appsody or Odo
+
+If you are running Codewind locally, to import your project it must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/).
+
+If you are running Codewind remotely on Kubernetes, to import your project it must include a Dockerfile and a Helm chart. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/) and for more information about Helm charts, see [Helm chart documentation](https://helm.sh/docs/topics/charts/).
```

Comment:
I think that using a statement such as the below will highlight this to a user?
`Remote projects will be built and run on Kubernetes. Your project must include a Dockerfile...`

Reply:
Reworked